internal/controller/apiextensions/composite/reconciler.go

```diff
@@ -527,7 +552,7 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 	// * Report which resources are not ready.
 	// * If a resource becomes Unavailable at some point, should we still report
 	//   it as Creating?
-	if ready != len(refs) {
+	if ready != len(refs) || renderFail {
```

Comment:
Won't `ready != len(refs)` always be true if `renderFail` is true? Or are there cases where we need both?

Reply:
You are correct :+1: `renderFail` was leftover from a previous iteration, removing now